pkg/controller/operator/master/reconciler.go

```diff
@@ -201,18 +201,25 @@ func (r *Reconciler) reconcileConfigMaps(ctx context.Context, config *operatorv1
 func (r *Reconciler) reconcileSecrets(ctx context.Context, config *operatorv1alpha1.KubermaticConfiguration, logger *zap.SugaredLogger) error {
 	logger.Debug("Reconciling Secrets")
 
-	creators := []reconciling.NamedSecretCreatorGetter{
-		common.WebhookServingCASecretCreator(config),
-		common.WebhookServingCertSecretCreator(config, r.Client),
-		common.ExtraFilesSecretCreator(config),
+	creatorsOrdered := [][]reconciling.NamedSecretCreatorGetter{
+		{
+			common.WebhookServingCASecretCreator(config),
```

Comment:
There is a special case when ensuring objects concurrently. CA's are expected to be created before Certs.